src/lib/load/load-files.ts

```diff
@@ -39,14 +39,15 @@ export default async function loadFiles(dir: string) {
       await Promise.all(filesToUpload.map(async asset => {
         const fileName = asset.filename_disk
         const assetPath = path.resolve(dir, 'assets', fileName)
-        const fileStream = new Blob([readFileSync(assetPath)], {type: asset.type})
+        const fileStream = new Blob([readFileSync(assetPath)], { type: asset.type })
 
         const form = new FormData()
         form.append('id', asset.id)
 
         if (asset.title) form.append('title', asset.title)
         if (asset.description) form.append('description', asset.description)
         if (asset.folder) form.append('folder', asset.folder)
+        if (asset.type) form.append('type', asset.type)
```

Comment:
This is the line that fixes the error I was facing when applying the AgencyOS template. See screenshot:

<img width="1367" height="698" alt="image" src="https://github.com/user-attachments/assets/39d6e36e-d06d-449e-ac0e-1579155a26da" />

In text:

```text
 ›   Warning: Directus Error: Invalid payload. "type" is required. (Status: 400) {"reason":"\"type\" is 
 ›   required","code":"INVALID_PAYLOAD"}
-- Request failed with status 400. Not retrying. Directus Error: Invalid payload. "type" is required. (Status: 400) {"reason":"\"type\" is required","code":"INVALID_PAYLOAD"}

```